protos/sources/blockexplorer/blockexplorer.proto

```diff
@@ -2,42 +2,75 @@ syntax = "proto3";
 
 package blockexplorer.api.v1;
 
-option go_package = "code.vegaprotocol.io/vega/protos/blockexplorer/api/v1";
 import "vega/commands/v1/transaction.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+option go_package = "code.vegaprotocol.io/vega/protos/blockexplorer/api/v1";
+
+option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
+  info: {
+    title: "Vega block explorer APIs";
+    version: "v0.59.0+dev";
+  }
+  schemes: [HTTP, HTTPS]
+  host: "lb.testnet.vega.xyz"
```

Comment:
Isn't the load balancer gone?

Reply:
I think it is, but I didn't know what to point it to to be fair.

Reply:
I don't either, but if it's non-functional that probably won't help. Maybe n06?